src/FakeClient/GraphicsCaptureEventProcessor.h

```diff
@@ -0,0 +1,230 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef FAKE_CLIENT_GRAPHICS_CAPTURE_EVENT_PROCESSOR_H_
+#define FAKE_CLIENT_GRAPHICS_CAPTURE_EVENT_PROCESSOR_H_
+
+#include "CaptureClient/CaptureEventProcessor.h"
+#include "Flags.h"
+#include "OrbitBase/WriteStringToFile.h"
+
+namespace {
```

Comment:
I think that we tend to still nest anonymous namespaces inside the module's namespace (`orbit_fake_client`).

Reply:
Done